packages/ui/src/2_molecules/Overlay/Overlay.types.ts

```diff
@@ -2,20 +2,14 @@ import { Align, AlignVertical } from '../../types/tailwind';
 
 export enum OverlayBackground {
   transparent = 'transparent',
-  light25 = 'light25',
-  light75 = 'light75',
-  dark25 = 'dark25',
-  dark75 = 'dark75',
+  default = 'default',
 }
 
 export const OverlayBackgroundClassName: {
   [key in OverlayBackground]: string;
 } = {
   [OverlayBackground.transparent]: 'bg-transparent',
-  [OverlayBackground.light25]: 'bg-gray-9 bg-opacity-25',
-  [OverlayBackground.light75]: 'bg-gray-9 bg-opacity-75',
-  [OverlayBackground.dark25]: 'bg-gray-1 bg-opacity-25',
-  [OverlayBackground.dark75]: 'bg-gray-1 bg-opacity-75',
+  [OverlayBackground.default]: 'bg-gray-70 bg-opacity-[0.69]',
```

Comment:
```suggestion
  [OverlayBackground.default]: 'bg-gray-70-ol',
```
I was looking at the design and you should use `rgba(44, 48, 59, 0.65)` for the default background. We already have that defined, it's `gray-70-ol` so you don't need to do it manually.